frontend/src/app/module-blueprint/components/dialogs/dialog-browse/dialog-browse.component.css

```
@@ -16,7 +16,7 @@
   margin-bottom: 10px;
   padding: 10px;
 
-  border-radius: 3px;
+  border-radius: 0;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)